src/core/test/system/system.t.sol

```diff
@@ -15,194 +15,93 @@
 
```

Comment:
The current systems are basically testing the borrower modules.

A method called `setupCurrencyOnLender` guarantees that the lender has currency for the tests